pages/episodes/[slug].vue

```diff
@@ -0,0 +1,39 @@
+<script setup>
+import { onMounted, ref } from 'vue'
+import { formatDate } from '~/utilities/helpers'
+import axios from 'axios'
+import Skeleton from 'primevue/skeleton'
+const dataLoaded = ref(false)
+const episode = ref([])
+const route = useRoute()
+onMounted(async () => {
+  await axios
+    .get(
+      `https://private-anon-c9c388aa36-nyprpublisher.apiary-proxy.com/api/v3/story/${route.params.slug}/`
+    )
+    .then((response) => {
+      episode.value = response.data.data.attributes
+      dataLoaded.value = true
+    })
+})
+</script>
+
+<template>
+  <div>
+    <div v-if="dataLoaded" class="episode container">
```

Comment:
should we being using a v-card here?

Reply:
we should use a VImageWithCaption or VSimpleResponsiveImage here... I'll take care of it.